excel.go

```diff
@@ -48,37 +126,34 @@ func getHeaderToXCoord(headerRow []string) map[string]int {
 	return headerToXCoord
 }
 
-type usernameAndPassword struct {
-	Username, Password string
-}
-
-func getMACFinUsers(f *excelize.File, input *Input) ([]usernameAndPassword, error) {
-	rows, err := f.GetRows(input.SheetName)
+func getMACFinUsers(f *excelize.File, input *Input, env Environment) (map[string]PasswordRow, error) {
```

Comment:
I discussed with Nazib. He said that it is OK if the last username/password is used.

Reply:
I added a function to remove duplicate MACFin users; in the case of duplicates, it saves the last user and deletes the others.

Reply:
I found a lot of small things after setting rowOffset to a value > 1. By setting it to `2`:
- I discovered a few places that were referencing a hardcoded value, rather than rowOffset
- the combination of rowOffset > 1 and an empty sheet caused the program to panic (slice out of bounds)
- - I now check for this case in the `validateSheets` function
- - I also added a few test cases for rowOffset > 1

Reply:
These changes will be put into a separate PR.